### ISSUE TITLE
Minecraft Bedrock Edition Crash After Restore

### ISSUE THREAD

**Issue by Yolotroll101** [FlagBrew/Checkpoint#146]
* [ ] Issue with Checkpoint for 3DS
* [x] Issue with Checkpoint for Switch
* [ ] Feature request
 
* Entrypoint: Paperclip jig for RCM with [CTCaer's Hektate Mod Payload](https://goo.gl/cfnRtr)
* System version: 5.1.0
* Checkpoint version: 3.4.0
* Expected behaviour: The game should start up with the modified save data present
* Encountered behavior: Upon startup, the game crashes immediately
* Result code (if available): --The crash is the game, not checkpoint. (Also the switch does not crash, just the game)
* Steps to reproduce:
  * Backup Minecraft data
  * -Optional- Modify data
  * Restore the data
  * Lanuch Minecraft

So basically, when I restore save data (Modified, or not) into minecraft, Also, this version crashes when I restore data while data exists, so I use [this method](https://github.com/FlagBrew/Checkpoint/issues/114#issuecomment-406903144) (fennectech's 3 posts) to do it.

**Comment by fennectech** [comment]
You have an old save that is broken    any saves created before today will not work.

**Comment by fennectech** [comment]
also   you need to update to 3.4.2 wich launched today.

**Comment by fennectech** [comment]
1.  Create a backup of your broken save
2. reset your savedata in the switches settings
3. launch minecraft and then create a world.
4. use checkpoint to backup this new save.
5. transfer the folders in /switch/Checkpoint/saves/0x0100D71004694000%20Minecraft/[oldsave]/games/com.mojang/minecraftWorlds
to 
/switch/Checkpoint/saves/0x0100D71004694000%20Minecraft/[newsave]/games/com.mojang/minecraftWorlds
6. open checkpoint and restore the new save
7. go see if minecraft launches  
 if it doesnt youll have to transfer the wrolds one at a time and restore the save to see wich one is borked.

**Comment by Yolotroll101** [comment]
So, update checkpoint, then fix. Got it, thanks. I'll let you know if I am successful.
